### ISSUE TITLE
Crash when using Gammaray with a debug windows build

### ISSUE THREAD

**Issue by rolivav** [KDAB/GammaRay#589]
When using Gammaray in debug I get this assert continuously in `PropertySyncer::handlerMessage` in this part of the code:
```
Message msg(m_address, Protocol::PropertyValuesChanged);
        msg << addr << (quint32)values.size();
        for (const auto &value : qAsConst(values))
            msg << value.first << value.second;
        emit message(msg);
```
it is specifically in the << operator. It seems it fails when saving the type `Qt::CaseSensitivity`, which is strange as this is probably a type already registered by Qt itself.

Here you can see the message reported by Messages window:
![image](https://user-images.githubusercontent.com/1664301/72277677-a1db7400-3632-11ea-8c6b-9cac4b2ef6cb.png)

If you ignore the assert long enough you get a crash and this stacktrace.
```
handleMessage (E:/GammaRay/core/tools/messagehandler/messagehandler.cpp:80)
QFSFileEngine::copy
QFSFileEngine::copy
QFSFileEngine::copy
QFSFileEngine::copy
QFSFileEngine::copy
QFSFileEngine::copy
GammaRay::Message::operator<<<QVariant> (E:/GammaRay/common/message.h:102)
GammaRay::PropertySyncer::handleMessage (E:/GammaRay/common/propertysyncer.cpp:141)
GammaRay::PropertySyncer::qt_static_metacall (E:/GammaRay/build/multibuild/GammaRay-Debug/common/gammaray_common_autogen/EWIEGA46WW/moc_propertysyncer.cpp:90)
QFSFileEngine::copy
QFSFileEngine::copy
GammaRay::Endpoint::dispatchMessage (E:/GammaRay/common/endpoint.cpp:362)
GammaRay::Server::messageReceived (E:/GammaRay/core/remote/server.cpp:213)
GammaRay::Endpoint::readyRead (E:/GammaRay/common/endpoint.cpp:169)
QtPrivate::FunctorCall<QtPrivate::IndexesList<>,QtPrivate::List<>,void,void (__cdecl GammaRay::Endpoint::*)(void)>::call (C:/Qt/5.12.3/msvc2017_64/include/QtCore/qobjectdefs_impl.h:152)
QtPrivate::FunctionPointer<void (__cdecl GammaRay::Endpoint::*)(void)>::call<QtPrivate::List<>,void> (C:/Qt/5.12.3/msvc2017_64/include/QtCore/qobjectdefs_impl.h:186)
QtPrivate::QSlotObject<void (__cdecl GammaRay::Endpoint::*)(void),QtPrivate::List<>,void>::impl (C:/Qt/5.12.3/msvc2017_64/include/QtCore/qobjectdefs_impl.h:415)
QFSFileEngine::copy
QFSFileEngine::copy
QFSFileEngine::copy
QFSFileEngine::copy
QHostAddress::isEqual
QHostAddress::isEqual
QHostAddress::isEqual
QHostAddress::isEqual
QHostAddress::isEqual
QDirModel::filter
QDirModel::filter
Application::notify 
QFSFileEngine::copy
QFSFileEngine::copy
QFSFileEngine::copy
CallWindowProcW
DispatchMessageW
QFSFileEngine::copy
FT_Property_Set
QFSFileEngine::copy
QFSFileEngine::copy
QFSFileEngine::copy
QOpenGLFunctions_4_2_Compatibility::glPixelStorei
QDirModel::filter
Application::run 
```
any idea what could be wrong?

**Comment by rolivav** [comment]
Hi, is GammaRay development abandoned?

**Comment by krf** [comment]
Definitely not. We're actively developing it, your issue just hasn't got any real attention so far.

I believe your problem is resolved by commit 16bf267435e5834834fc765c7860d0c93ea490de.

Could you check GammaRay from Git master?
